vital/types/feature_track_set.h

```diff
@@ -65,24 +65,24 @@ class VITAL_EXPORT feature_track_state : public track_state
 public:
   //@{
   /// Constructor
-  explicit feature_track_state( frame_id_t frame,
-                                feature_sptr const& feature = nullptr,
-                                descriptor_sptr const& descriptor = nullptr,
-                                bool inlier = false )
-    : track_state{ frame }
-    , feature{ feature }
-    , descriptor{ descriptor }
-    , inlier{ inlier }
+  explicit feature_track_state( frame_id_t p_frame,
```

Comment:
So... this is inconsistent with other places that renamed e.g. "foo" → "foo_". (FWIW, I prefer the "p_foo" convention, i.e. here.)